umake/__init__.py

```diff
@@ -116,6 +125,9 @@ def main():
     parser.add_argument("-v", "--verbose", action="count", default=0, help=_("Increase output verbosity (2 levels)"))
 
     parser.add_argument('-r', '--remove', action="store_true", help=_("Remove specified framework if installed"))
+    parser.add_argument('-l', '--list', action="store_true", help=_("List all frameworks"))
+    parser.add_argument('--list-installed', action="store_true", help=_("List installed frameworks"))
+    parser.add_argument('--list-available', action="store_true", help=_("List installable frameworks"))
```

Comment:
Oh, just one more thing here.
Those options are uncompatible (you are using one or the other). So you need to create a `add_mutually_exclusive_group()` and add them to it, so that argparse will enable to only use one of them.

Look at runtests, I'm using such an example here.